### DISCUSSION TITLE
SBAS

### DISCUSSION THREAD

**Discussion by wanghengsoft** [gmtsar/gmtsar#749] [General]
dimension don not match  ../intf_all/2015020_2015068/corr.grd,  when I  use command sbas

**Comment by Xiaohua-Eric-Xu** [comment]
Did you cut your data running snaphu.csh?

**Comment by wanghengsoft** [comment]
Thank you for your reply.&nbsp;I have solved this problem，Maybe this command line "# set xdim = `gmt grdinfo -C ../intf_all/2015068_2015092/unwrap.grd | awk '{print $10}'`# set ydim = `gmt grdinfo -C ../intf_all/2015068_2015092/unwrap.grd | awk '{print $11}'`"&nbsp; is invalid ,&nbsp; I edit this line "sbas intf.tab scene.tab 9 5 $xdim $ydim -smooth 1.0 -wavelength 0.0554658 -incidence 30 -range 800184.946186 -rms -dem" ,replaced "$xdim $ydim" with "1348 1371"




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "gmtsar/gmtsar"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月16日(星期五) 上午10:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [gmtsar/gmtsar] SBAS (Discussion #749)





 
Did you cut your data running snaphu.csh?
